addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -111,7 +139,7 @@ sAnimations.registry.ProductWishlist = sAnimations.Class.extend(ProductConfigura
     /**
      * @private
      */
-    _removeWish: function (e, deferred_redirect){
+    _removeWish: function (e, deferred_redirect) {
```

Comment:
Should not be done in 12.0 for no reason. Will only make CHS hates us more.

Reply:
ok